src/assets/i18n/fr.json

```diff
@@ -348,10 +305,22 @@
 	"Remove color": "Supprimer la couleur",
 	"Change color midpoint": "Changer le point d'interpolation de la palette?",
 	"Back to templates": "Retour aux palettes de couleurs prédéfinies",
+	"Edit a layer": "Editer une couche cartographique",
+	"Edit a visualisation set": "Editer un Espace de visualisation",
 	"Add layer": "Ajouter une couche de données",
+	"layer description": "Une couche cartographique vous permet d'analyser une donnée géographique présente dans la collection. Vous pouvez personnaliser la donnée géographique à afficher et le rendu de cette donnée.",
 	"#": "#",
+	"Preview": "Prévisualiser",
+	"Edit": "Modifier",
 	"At least one layer is required": "Au moins une couche de données est nécessaire",
+	"Add visualisation": "Ajouter un Espace de visualisation",
+	"visualisation set description": "Un espace de visualisation est destiné à y placer des couches cartographiques qui seront affichées/masquées ensemble.",
+	"Layers": "Couches de données",
+	"Displayed by default": "Affiché par défaut",
+	"At least one visualisation set is required": "",
 	"The field is mandatory.": "Le champs est obligatoire",
+	"All layers": "",
+	"At least one visualisation set should be assigned to this layer": "La couche cartographique doit être placée dans, au moins, un Espace de travail.",
```

Comment:
un Espace de visualisation (not travail) ==> my bad it's my fault

Reply:
Fixed